packages/vite-node/rollup.config.js

```diff
@@ -32,6 +32,9 @@ const external = [
   'node:events',
   'node:vm',
 ]
+const esmOnly = [
```

Comment:
It needs to be moved to `devDependencies` to be bundled.

Reply:
fixed

Reply:
> It needs to be moved to `devDependencies` to be bundled, not just filtered. Otherwise consumers of CJS package will install a package that's not being used

Just check the bundle artifacts, it is ok, because `rollup-plugin-esbuild` bundle it, and  the esm build still use the original dependency, only cjs build use the bundled version.

But move to `devDependency` maybe a better approach.